src/main/java/org/folio/service/finance/budget/BudgetRestrictionService.java

```diff
@@ -61,6 +61,7 @@ public void checkEncumbranceRestrictions(List<? extends EncumbranceRelationsHold
   }
 
   private boolean isRemainingAmountExceed(Budget budget, MonetaryAmount newEncumberedAmount) {
+    // im check
```

Comment:
Following check looks already updated

Reply:
This is pr for the similar method of mod-invoice https://github.com/folio-org/mod-invoice/pull/500/files#diff-c800384a86f17f7278954f133f940d3e241d4869af4ec9671d6147ca31e8efd4R79-R97
you can use as example in here. Please check the logic one more time